ui/src/common/storage.js

```diff
@@ -26,8 +43,8 @@ const storage = {
   getSync(key) {
     // eslint-disable-next-line react/no-this-in-sfc
     if (this.isAvailable) {
-      const rawValue = localStorage.getItem(key);
-      return JSON.parse(rawValue);
+      const rawValue = localforage.getItem(key).then(item => JSON.parse(item));
+      return rawValue;
```

Comment:
This will return undefined rawValue 🤔 why not use async?

Reply:
Also according to the documentation:

> [...] localForage automatically does JSON.parse() and JSON.stringify() when getting/setting values.

Reply:
Changed to use asynchronous syntax and removed json parsing